sdk/samples/iot/hub/src/pnp_helper.h

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PNP_HELPER_H
+#define _az_PNP_HELPER_H
+
+#include <stdint.h>
+
+#include <azure/core/az_json.h>
+#include <azure/core/az_result.h>
+#include <azure/core/az_span.h>
+#include <azure/iot/az_iot_hub_client.h>
+
+#include "sample_pnp_thermostat_component.h"
+
```

Comment:
this is wrong dependency .. helper should not depend upon sample

Reply:
you're right thank you